libs/externals.liq

```diff
@@ -127,15 +127,15 @@ def enable_external_flac_decoder() =
   if file.which(flac) != "" then
     log(label="external.decoder","Enabling EXTERNAL_FLAC external decoder.")
     flac_p = "#{flac} -d -c -"
-    def test_flac(file) =
```

Comment:
Does that mean that record variables cannot be overridden?

Reply:
You can override as usual. But if you define
```
file = "toto"
```
then `file` is precisely `"toto"` (you lose all the fields), so that you cannot use `file.which` anymore: this was the problem here. We could do things in another way, but that the most simple one.

Reply:
Word.